packages/apis/ipfs/README.md

```diff
@@ -0,0 +1,45 @@
+# IPFS Web3API package
+
+## Functions
+
+### Query
+
+`catFile(ipfsUrl: String!, cid: String!): Bytes!`
+
+Returns byte content of file, defined by provided _cid_.
```

Comment:
These sorts of comments can be inline in the schema, instead of in the README. Ideally the README just points to the schema so that way it's "self documenting" as changes are made.